extensions/amp-twitter/1.0/component.js

```diff
@@ -86,19 +103,48 @@ function BentoTwitterWithRef(
     ]
   );
 
+  const observerCb = useIntersectionObserver(({isIntersecting, target}) => {
+    if (isIntersecting) {
+      setinView(true);
+
+      /**
+       * Get Main bento-twitter reference. Used the alternative way here to
+       * get the <bento-twitter> reference as using offsetParent is forbidden.
+       */
+      bentoTwitterRef.current = target.parentNode.parentNode.host;
```

Comment:
Can you explain why we need to set bentoTwitterRef.current here? Will target.parentNode.parentNode.host always be set?

Reply:
The `bentoTwitterRef` would be used in `messageHandler` to resize the bento component with the actual height the embed would need using the `setComponentHeight(height)` callback. 
> Will target.parentNode.parentNode.host always be set?

Yes outside the storybook the `target.parentNode.parentNode.host` will always be set in both AMP and bento context. It may throw a console error in the storybook that won't break the storybook. We have used this alternative way to get the reference to the `<bento-twitter>` tag as using the offsetParent is forbidden.

Reply:
Please do let me know if you have any suggestions for a better alternative way to get the `<bento-twitter>` element reference to set height.

Reply:
👍🏼

Reply:
maybe just add a line to the doc block explaining that this is used for Storybook, otherwise it looks odd

Reply:
We need this in the normal html page when using `<bento-twitter>` component to resize the embed as it uses the shadow dom.